sdk/recoveryservices/Microsoft.Azure.Management.RecoveryServices/src/generate.ps1

```diff
@@ -0,0 +1 @@
+Start-AutoRestCodeGeneration -ResourceProvider "recoveryservices/resource-manager" -AutoRestVersion "latest"
```

Comment:
Hi, we noticed the latest autorest doesn't work well with c# generator, so please use V2 and regenerate the SDK. Thanks
```suggestion
Start-AutoRestCodeGeneration -ResourceProvider "recoveryservices/resource-manager" -AutoRestVersion "V2"
```

Reply:
please also delete `sdk/recoveryservices/Microsoft.Azure.Management.RecoveryServices/src/Generated/code-model-v1`, which is created by the latest autorest, which it shouldn't